### PR TITLE
Refactor/about-us

### PR DESCRIPTION
**Pull request by ashtotakoe**
##### Link to Jira task
NO-JIRA
##### What kind of change does this PR introduce?

- [x] Bug fix
- [ ] Feature/change request
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build or CI related changes
- [ ] Other (include details)

##### What has changed?
- minor info changes
- add missing collaborator

##### Other information
- None

### CODE REVIEW

**Review by ADyBaH — APPROVED** [review]

**Review by DeePenguin — CHANGES_REQUESTED** [review]
We agreed to add info about our mentor in separate block, which also was supposed to include general information about our team and working process

**Review by DeePenguin — CHANGES_REQUESTED** [review]

File: src/app/about/about-page.component.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <div class="container">
   <h2 class="header tui-text_h4">About us</h2>
+  <ec-general-info />
+  <h2 class="header tui-text_h4">Our team</h2>
```
[the remark this answers is not in the document]
Comment (ashtotakoe) [reply src/app/about/about-page.component.html]: fixed

File: src/app/about/components/general-info/general-info.component.html
[the remark this answers is not in the document]
Comment (ashtotakoe) [reply src/app/about/components/general-info/general-info.component.html:1]: done!

File: src/app/about/components/general-info/general-info.component.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core'
+
+@Component({
+  selector: 'ec-general-info',
+  templateUrl: './general-info.component.html',
+  styleUrls: ['./general-info.component.scss'],
```
[the remark this answers is not in the document]
Comment (ashtotakoe) [reply src/app/about/components/general-info/general-info.component.ts:6]: done!

**Review by DeePenguin — APPROVED** [review]

**Review by ADyBaH — APPROVED** [review]

**Pull request merged**